### PR TITLE
Chatbot added successfully

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#128 
Closes: # (issue number)

## Proposed changes

Brief description of what is fixed or changed
Chatbot added successfully.Initially basic commands like hi, how are you ? , good morning , good night and help. More commands can be added as per requirement.The target needs to send the message stating with"/" and then the chatbot is automatically activated and replies goes itself to the target.
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot from 2020-05-08 01-14-39

WhatsApp Image 2020-05-08 at 1 16 29 AM

## Other information

Any other information that is important to this pull request
I want to add few more features like doing google search or images So I would request to keep this issue open.Initial files have been added successfully.

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/chatbot.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5489551&bid=17989615)

**Comment by SinghHimadri** [comment]
@rpotter12 @xandao6 Please review my PR.

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/chatbot.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5489551&bid=17989615)

File: wplay/chatbot.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        print("Command args: {cmd}".format(cmd=command_args))
+        if len(command_args) == 1 and command_args[0] in simple_menu_keys:
+            return simple_menu[command_args[0]]()
+
+        else:
+            return "Wrong command. Send me /help to see a list of valid commands"
+
+    except KeyError as e:
+            print("Key Error Exception: {err}".format(err=str(e)))
+
```
Comment (rpotter12) [inline wplay/chatbot.py:24]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5489551)

**Comment by SinghHimadri** [comment]
> @SinghHimadri What happens if the simple_menu is not having the last_message??
> 
> I think you should use try command in this

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/chatbot.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5489551&bid=17989615)

File: wplay/chatbot.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        command_args = last_Message[1:].split(" ", 1)
+        print("Command args: {cmd}".format(cmd=command_args))
+        if len(command_args) == 1 and command_args[0] in simple_menu_keys:
+            return simple_menu[command_args[0]]()
+        else:
+            return "Wrong command. Send me /help to see a list of valid commands"
+    except KeyError as e:
+            print("Key Error Exception: {err}".format(err=str(e)))
+            
+
```
Comment (rpotter12) [inline wplay/chatbot.py:22]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5489551)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/chatbot.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5489551&bid=17989615)

**Comment by SinghHimadri** [comment]
> @SinghHimadri What happens if the simple_menu is not having the last_message??
> 
> I think you should use try command in this

I have added try except block and bot replies back to target to use /help command.
Have a look.

Screenshot from 2020-05-08 16-37-23

WhatsApp Image 2020-05-08 at 4 35 37 PM

**Comment by xandao6** [comment]
Well, it will be our own chatbot. We will need a local database, it's hard.

**Comment by SinghHimadri** [comment]
> Well, it will be our own chatbot. We will need a local database, it's hard.

Actually @rpotter12 said that it should be able to few commands that I have already added.

**Comment by rpotter12** [comment]
> Well, it will be our own chatbot. We will need a local database, it's hard.

For now it is acceptable as inial script. For next PR we will aspect the full working.

**Comment by SinghHimadri** [comment]
@rpotter12 Please add labels. Thank you.

**Pull request merged**